rskj-core/src/main/java/co/rsk/peg/storage/BridgeStorageAccessor.java

```diff
@@ -1,55 +1,24 @@
 package co.rsk.peg.storage;
 
 import co.rsk.core.RskAddress;
-import org.ethereum.core.Repository;
 import org.ethereum.vm.DataWord;
 import org.ethereum.vm.PrecompiledContracts;
 
 import java.io.IOException;
 
-public class BridgeStorageAccessor {
+public interface BridgeStorageAccessor {
 
-    private final Repository repository;
-    private static final RskAddress contractAddress = PrecompiledContracts.BRIDGE_ADDR;
+    RskAddress contractAddress = PrecompiledContracts.BRIDGE_ADDR;
```

Comment:
I wouldn't put variables in an interface. It's a contract that exposes certain methods that allow access to the storage.

What the contract address is, I think that is related to the concrete implementation

Reply:
good point. Maybe we should rename it to `StorageAccessor`, since it won't have references to the bridge